src/annotator/config/index.js

```diff
@@ -19,6 +19,7 @@ function configFrom(window_) {
 
     annotations: settings.annotations,
     branding: settings.hostPageSetting('branding'),
+    onLayoutChange: settings.hostPageSetting('onLayoutChange'),
```

Comment:
I wonder if we should rename this to `onSidebarLayoutChange` to be more explicit about what thing's layout changed?